src/Page/Race/Details.elm

```diff
@@ -108,16 +108,20 @@ resultsTable results riders signedIn =
             resultCategories
 
 
+resultToInt : RaceResult -> Int
+resultToInt =
+    .result
+        >> String.toInt
+        >> Result.withDefault (floor (1 / 0))
```

Comment:
1 / 0 is infinity en dan met floor maakt die er een integer van? Niet heel erg straight-forward. Nadeel is trouwens ook dat alle values met characters dan in willekeurige volgorde getoond worden en niet alfabetisch.

Reply:
Fixed in volgende commit.